### PR TITLE
Change repo name

### PR DESCRIPTION
**Pull request by jaelim-ms**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# MobileCenter-SDK-iOS Change Log
+# mobile-center-sdk-ios Change Log
```
Comment (lumaxis) [inline CHANGELOG.md]: You could use the proper SDK name "Mobile Center SDK for iOS" here instead of the repo name. Looks a bit nicer imo 🙂

Comment (jaelim-ms) [inline CHANGELOG.md]: I agree.  😄

Comment (clpolet) [inline CHANGELOG.md]: Erg I though this one was already fixed, not a big deal though. Will be done in another PR.
